core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorAccessControl.java

```diff
@@ -469,6 +474,17 @@ default void checkCanRenameMaterializedView(ConnectorSecurityContext context, Sc
         denyRenameMaterializedView(viewName.toString(), newViewName.toString());
     }
 
+    /**
+     * Check if identity is allowed to grant an access to the function execution to grantee.
+     *
+     * @throws AccessDeniedException if not allowed
+     */
+    default void checkCanGrantExecuteFunctionPrivilege(ConnectorSecurityContext context, FunctionKind functionKind, SchemaRoutineName functionName, TrinoPrincipal grantee, boolean grantOption)
```

Comment:
There was no `checkCanGrantExecuteFunctionPrivilege` in `ConnectorAccessControl` at all?

Reply:
Previously the functions were global, PTFs are connected to catalogs